.github/ISSUE_TEMPLATE/feature-issue.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Feature Issue
+about: Feature Issue
+title: ''
+labels: feature
+assignees: ''
+
+---
+
+# Feature Issue
+
+[comment]: # (Please replace ... with your information. Remove < and >)
+
+### User Story
+
+As a <user|developer|...>, I want to <task> **so that** <goal>. 
+
+### Description
+
+[comment]: # (Describe the feature you would like and include designs and Figma links if available.)
+*Summary*: ...
+
```

Comment:
@jakubgs can you please add:

### Uses cases
[comment]: # (Describe the use cases the implementation should cover. E.g. New users, existing users, public chat, group chat, etc)

Reply:
Done.